flagd/pkg/service/flag-evaluation/connect_service.go

```diff
@@ -77,7 +77,7 @@ func NewConnectService(
 		logger:  logger,
 		eval:    evaluator,
 		metrics: mRecorder,
-		eventingConfiguration: &eventingConfiguration{
+		eventingConfiguration: &EventingConfiguration{
```

Comment:
You may parameterize this to `NewConnectService()` so that the connect service works with an implementation of an interface.

Reply:
This introduced some churn without much immediate need to the flagd codebase. I'd like to skip this for now, unless you feel strongly about it.